lib/take_away.rb

```diff
@@ -0,0 +1,48 @@
+require_relative './menu'
+require_relative './send_sms'
+
+class TakeAway
+  attr_reader :cart
+
+  def initialize(menu: Menu.new, sender: SmsSender.new)
+    @menu = menu
+    @sender = sender
+    @cart = []
+  end
+
+  def menu
+    @menu.list_dishes
+  end
+
+  def add_to_cart(name, quantity = 1)
```

Comment:
Good use of single responsibility principle. You are splitting out each component into its own method.